contents/blog/posthog-vs-ga4.md

```diff
@@ -0,0 +1,536 @@
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Formulas</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Use custom formulas to create unique insights</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>SQL access</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Write your own queries in SQL</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Toolbar</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">View insights on your live website or app with an overlay</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Apps and integrations
+
+Integrations are another major point of difference between GA4 and PosthHog, especially when it comes to importing data. Because PostHog is built to be your single source of truth for viewing data, it's very simple to import data from other sources. 
+
+On the other hand, it's hard to import data into GA4 because:
+
+1. The type of data you're allowed to [import into GA4](https://support.google.com/analytics/answer/10071301) is limited. 
+2. You need to constantly upload CSV files manually, or you need to set up an SFTP server to automatically do this for you.
+
+However, GA4 is well integrated into the Google ecosystem, making it easy to transfer data between it and other Google products.
+
+Below is a comparison of some of the most popular apps. For a full list of PostHog’s available integrations, [check our app directory](/cdp).
+
+<div className="overflow-x-auto -mx-5 px-5 w-screen md:w-auto">
+  <table className="w-full mt-4">
+    <thead>
+      <tr>
+        <td className="w-3/12"></td>
+        <td class="text-center"><strong>GA4</strong></td>
+        <td class="text-center"><strong>PostHog</strong></td>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><p class="!mb-0 pb-0.5 !leading-tight"><strong>Zapier</strong></p><p class="!mb-0 !text-sm text-opacity-75 leading-none">Export events for use in Zaps.</p></td>
```

Comment:
Is this the number one Zapier use case?

"Trigger Zaps with events" instead maybe?